source/dev_testing.md

```diff
@@ -1,44 +1,80 @@
-# Integration Testing
+# Integration+ Testing
 
-Odahu uses the robotframework for an integration testing.
-System/integration tests located in the following directories:
-* `packages/robot` - a python package with additional Robot libraries. For example: kubernetes, auth_client, feedback, and so on. 
-* `packages/tests/stuff` - artifacts for integration testing. For example: pre-trained ML artifacts, test toolchain integrations, and so on.
-* `packages/tests/e2e` - directory with the RobotFramework tests.
+This page provides information about testing of ODAHU.
+ODAHU uses [Robot Framework](https://robotframework.org/) for an integration, system and end-to-end testings.
 
-## Running system/integration tests
+All tests are located in the following directories of the [ODAHU project](https://github.com/odahu/odahu-flow):
+* `packages/robot/` - a python package with additional Robot libraries. For example: kubernetes, auth_client, feedback, and so on. 
+* `packages/tests/stuff/` - artifacts for integration testing. For example: pre-trained ML artifacts, test toolchain integrations, and so on.
+* `packages/tests/e2e/` - directory with the RobotFramework tests.
+
+## Preparing for testing
+--------------------
+
+It's expected that you are using POSIX-compliant operating system and have installed Python 3.6.9+ and pip.
```

Comment:
We really have dependency >=3.6.9 OR we can set 3.6>= as a dependency? Because the requirement to have last minor version could be a bit confusing for documentation/system users

Reply:
if we indicate 3.6+ nobody will try to run it on Python 3.6.0?
Because I use 3.6.9 and I didn't install Python 3.6.0 and checked.